### PR TITLE
 WiFi RSSI Retrieval and update README for Clarity 

### PR DESCRIPTION
**Pull request by ToledoEM**
# Pull Request: Update README for Clarity and WiFi RSSI Retrieval

## Description

This update improves the README file by providing clearer instructions on the configuration for HA.

## Changes

* Updated README to reflect accurate information on retrieving WiFi RSSI from the API
* Added example code snippets for clarity

## Impact

With this change, users can now easily access and utilize the WiFi RSSI data provided by the API.

image

### CODE REVIEW

File: .DS_Store
Comment (ms32035) [inline .DS_Store:1]: Please remove this file

File: .gitignore
Comment (ms32035) [inline .gitignore:1]: This should be part of your global `.gitignore` not this repo

File: custom_components/ubibot/const.py [hunk 1]
```diff
@@ -3,7 +3,9 @@
     DEVICE_CLASS_HUMIDITY,
     DEVICE_CLASS_ILLUMINANCE,
     DEVICE_CLASS_TEMPERATURE,
+    DEVICE_CLASS_SIGNAL_STRENGTH,
     TEMP_CELSIUS,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT
```
Comment (ms32035) [inline custom_components/ubibot/const.py]: Comma missing, please run the pre-commit hooks. You can upgrade them as well

File: custom_components/ubibot/const.py [hunk 2]
```diff
@@ -25,6 +27,12 @@
...
+    "wifi_rssi": {
+        "class": DEVICE_CLASS_SIGNAL_STRENGTH,
+        "unit": SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+        "icon": "mdi:wifi",
+        "field": "field5",
+    },
 }
 
-MODELS = {"ubibot-ws1": "WS1"}
+MODELS = {"ubibot-ws1": "WS1"}
```
Comment (ms32035) [inline custom_components/ubibot/const.py]: EOL/EOF should not be changed. I believe pre-commit should detect that

**Review by ms32035 — CHANGES_REQUESTED** [review]
Thank you for the contribution, I've added com comments

**Comment by ToledoEM** [comment]
Perfect, this is not my thing and now I learned what a pre-commit hook is. I see the value of them. 
Cool.

**Pull request merged**